x-pack/plugins/cases/public/containers/constants.ts

```diff
@@ -8,7 +8,7 @@
 import type { SingleCaseMetricsFeature } from './types';
 
 export const DEFAULT_TABLE_ACTIVE_PAGE = 1;
-export const DEFAULT_TABLE_LIMIT = 5;
+export const DEFAULT_TABLE_LIMIT = 10;
```

Comment:
@jonathan-buttner @cnasikas is there a reason why the largest number of cases we allow per page is 25? This seems quite low if you have 100s+ cases in total. For other tables, such as the alerts page, 50 and 100 are common options.

Reply:
We might run into performance concerns if we bump it too high since we do aggregations and other joins. I think 100 reasonable though.